README.md

```diff
@@ -130,6 +130,10 @@ These projects have been developed mainly in Toronto, or by Torontoians
 - [@UofTCoders](https://github.com/UofTCoders) - Peer learning group at the University of Toronto ([Website](https://uoftcoders.github.io/))
 - [Polyhack](https://www.meetup.com/polyhackTO/) - Polyglot technical meetup ([Website](https://zachgoldstein.github.io/polyhack/))
 
+## Companies that are based in Toronto
+
+- [Jam3](https://github.com/Jam3) - Design studio with many OSS contributions. [Website](https://www.jam3.com/)
```

Comment:
Since you moved `Jam3` from organizations to companies, I would suggest do the same thing with its markdown file also.

Reply:
It does!
Another question though, why did you put Companies section in lower part of README. Shouldn't it be more prominent? Something like this order:
- Projects
- Organizations
- Companies
- Users
- Incubators
- Other

Reply:
Because I think that the other sections are more important for a local community, I guess. I didn't think too much about it. :)

Reply:
Cool!